src/error.rs

```diff
@@ -1,7 +1,8 @@
 //! Error-handling module for the crate
 
 use crate::{
-    io::parser::ParseError, logical::types::LogicalTypeEnum,
+    io::parser::ParseError,
+    logical::{model::Term, types::LogicalTypeEnum},
     physical::datatypes::float_is_nan::FloatIsNaN,
```

Comment:
Following the [style guide for imports](https://doc.rust-lang.org/1.0.0/style/style/imports.html), we should reorder the imports as follows:
```rust
use thiserror::Error;

use crate::{
    io::parser::ParseError,
    logical::{model::Term, types::LogicalTypeEnum},
    physical::datatypes::float_is_nan::FloatIsNaN,
};
```